### DISCUSSION TITLE
How to change the request body from "multipart/form-data" to "application/json"?

### DISCUSSION THREAD

**Discussion by gabrielgoncalfreitas** [hussein4alaa/laravel-g4t-swagger-auto-generate#37] [General]
Hello!

I have a custom request class for my user registration method. Currently the package recognizes the Custom Request class created by `php artisan make:request` as **multipart/form-data**, but I want it to be recognized as **application/ json** on the **Accept** header option.

Pics from my code

Custom request:
![image](https://github.com/hussein4alaa/laravel-g4t-swagger-auto-generate/assets/67321807/8718e795-b6a2-487f-8322-3cfa6d613c31)

Controller method where the custom request is used:
![image](https://github.com/hussein4alaa/laravel-g4t-swagger-auto-generate/assets/67321807/c7c31605-6188-41d9-9049-c60c1b4100a0)

Swagger method doc: 
![image](https://github.com/hussein4alaa/laravel-g4t-swagger-auto-generate/assets/67321807/4726e84c-d8ed-4e29-89c7-30f77d3f3179)

**Comment by hussein4alaa** [comment]
Dear @gabrielgoncalfreitas,
use [3.0.7](https://github.com/hussein4alaa/laravel-g4t-swagger-auto-generate/releases/tag/3.0.7) version and you will find it available.
